flink-java/src/main/java/org/apache/flink/api/java/sca/TaggedValue.java

```diff
@@ -46,7 +50,10 @@
 		NULL // null
 	}
 
-	public static enum Input {
+	/**
+	 * Inputs types.
```

Comment:
This is a bit misleading, I believe these are used a tags to remember which input they belong to for 2 input operators (like coGroup for example). They may have the same type though ;)